### DISCUSSION TITLE
Want when click on new terminal should go to default SSH

### DISCUSSION THREAD

**Discussion by devesh288** [Eugeny/tabby#6487] [Q&A]
Whenever i am clicking on new terminal it is going to CMD only where i want it should go to ssh. Also evreytime i have to click on profile and create ssh connection , is it possible that new terminal ask details about ssh server and save in profile and mention under group?

Appreciate on reply.

thanks...

**Comment by fuzhou314** [comment]
I am now using version 1.0.227 and have this question, and the original intention of using tabby is to be able to ssh quickly. But turning on CMD by default is contrary to my original intention.
